### PR TITLE
FIX. Enable re-sending again - http://magento.stackexchange.com/a/376…

### PR DESCRIPTION
**Pull request by seansan**
…90/78

FIX. Enable re-sending again - http://magento.stackexchange.com/a/37690/78

Irritating "new" feature so order emails cannot be resent ... fixed

### CODE REVIEW

**Comment by LeeSaferite** [comment]
I think the better solution is to switch to `queueNewOrderEmail(true)`

**Comment by seansan** [comment]
As long as the problem is fixed ;P Can you create a pull request and add it?

**Comment by shirtsofholl** [comment]
Can this be accepted?

**Comment by diglin** [comment]
+1 for the @shirtsofholl solution

File: app/code/core/Mage/Adminhtml/controllers/Sales/OrderController.php [hunk 1]
```diff
@@ -136,6 +136,8 @@ public function emailAction()
     {
         if ($order = $this->_initOrder()) {
             try {
+                // FIX. Enable re-sending again - http://magento.stackexchange.com/a/37690/78
+				$order->setEmailSent(false);
```
Comment (colinmollenhour) [inline app/code/core/Mage/Adminhtml/controllers/Sales/OrderController.php]: Tabs vs spaces... We gotta have some standards.. :)

Comment (seansan) [inline app/code/core/Mage/Adminhtml/controllers/Sales/OrderController.php]: Agree .... +1 .. just get this one in ;)

Comment (colinmollenhour) [inline app/code/core/Mage/Adminhtml/controllers/Sales/OrderController.php]: Preferred way to fix is while on your PR branch just replace the tabs with spaces, use `git commit -a --amend` to fix the existing commit and then `git push -f` to force-push over the old commit.

**Comment by colinmollenhour** [comment]
Merged 1.9.2.3 into 1.9.3.0

**Pull request merged**